packages/core/src/Stores/portfolio-store.js

```diff
@@ -131,6 +134,12 @@ export default class PortfolioStore extends BaseStore {
         this.error = '';
         if (response.portfolio.contracts) {
             this.positions = response.portfolio.contracts
+                .filter(pos => {
```

Comment:
Can we break this out into a function called filterDisabledPositions or something relevent?